### ISSUE TITLE
NPE with Context Alert Filters in daemon mode

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#4405]
Issue reported in OWASP ZAP User Group: https://groups.google.com/d/topic/zaproxy-users/6kPcTT9RqeE/discussion

Steps to reproduce the issue:
1. Run ZAP in daemon mode (and with Context Alert Filters add-on);
2. Create a context;
3. Delete that context;
4. Proxy/send HTTP messages that would raise an alert
5. There should be some exceptions in the log/output:
```
187532 [ZAP-PassiveScanner] ERROR org.zaproxy.zap.extension.alertFilters.ExtensionAlertFilters  - Error handling alert
java.lang.NullPointerException
	at org.zaproxy.zap.extension.alertFilters.ExtensionAlertFilters.handleAlert(ExtensionAlertFilters.java:402)
	at org.zaproxy.zap.extension.alertFilters.ExtensionAlertFilters.handleAlert(ExtensionAlertFilters.java:372)
	at org.zaproxy.zap.extension.alertFilters.ExtensionAlertFilters.eventReceived(ExtensionAlertFilters.java:341)
	at org.zaproxy.zap.eventBus.SimpleEventBus.publishSyncEvent(SimpleEventBus.java:197)
	at org.zaproxy.zap.extension.alert.ExtensionAlert.publishAlertEvent(ExtensionAlert.java:287)
	at org.zaproxy.zap.extension.alert.ExtensionAlert.alertFound(ExtensionAlert.java:220)
	at org.zaproxy.zap.extension.pscan.PassiveScanThread.raiseAlert(PassiveScanThread.java:219)
	at org.zaproxy.zap.extension.pscanrulesAlpha.XPoweredByHeaderInfoLeakScanner.raiseAlert(XPoweredByHeaderInfoLeakScanner.java:135)
	at org.zaproxy.zap.extension.pscanrulesAlpha.XPoweredByHeaderInfoLeakScanner.scanHttpResponseReceive(XPoweredByHeaderInfoLeakScanner.java:67)
	at org.zaproxy.zap.extension.pscan.PassiveScanThread.run(PassiveScanThread.java:148)
```

Add-on:
Context Alert Filters version 6 (and older).

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#1377.

**Comment by thc202** [comment]
Released in version 7 of Context Alert Filters add-on.
